src/renderer/components/Editor/SquareBracket.tsx

```diff
@@ -1,44 +1,47 @@
+import { styled } from '@mui/material';
 import { Box } from '@mui/system';
-import { useState } from 'react';
 import colors from 'renderer/colors';
 import SquareBracketHover from './SquareBracketHover';
 
 interface Props {
   isLast: boolean;
   isTakeGroupOpened: boolean;
+  takeIndex: number;
+  takeGroupId: number;
 }
 
-const SquareBracket = ({ isLast, isTakeGroupOpened }: Props) => {
+const SquareBracket = ({
+  isLast,
+  isTakeGroupOpened,
+  takeIndex,
+  takeGroupId,
+}: Props) => {
   const bottomWidth = isLast || !isTakeGroupOpened ? '2px' : '0px';
 
-  const [isHovering, setIsHovering] = useState(false);
+  const componentClassName = `squareBracket_${takeGroupId}_${takeIndex}`;
```

Comment:
Probably make this an ID then if it's unique.

Also `['squareBracket', takeGroupId, takeIndex].join('-')` is cleaner (use `_` or `-` either is okay, `-` is more css-y though)